style/style.css

```diff
@@ -6,17 +6,33 @@
   padding: 0;
   box-sizing: border-box;
   text-decoration: none;
-  width: 375px;
-  height: auto;
   position: relative;
   left: 15px;
   top: 15px;
```

Comment:
- [Optional] I noticed you added a position relative to all your elements and this is making your whole page to be shifted to the left of the screen. Please consider adding the position relative to a class and assign that class to any element you want to instead. For example;
```css
.rel {
  position: relative;
  left: 15px;
  top: 15px;
}
```
Then in your HTML;
```html
<div class="rel">
  ...
</div>
```